features.json

```diff
@@ -2,77 +2,107 @@
 	"$schema": "./features.schema.json",
 	"features": {
 		"bigInt": {
-			"description": "JS BigInt to Wasm i64 integration",
+			"description": "JS BigInt to Wasm i64 Integration",
 			"url": "https://github.com/WebAssembly/JS-BigInt-integration",
 			"phase": 5
 		},
+		"branchHinting": {
+			"description": "Branch Hinting",
+			"url": "https://github.com/WebAssembly/branch-hinting/blob/master/proposals/branch-hinting/Overview.md",
+			"phase": 5
+		},
 		"bulkMemory": {
-			"description": "Bulk memory operations",
+			"description": "Bulk memory Operations",
 			"url": "https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md",
 			"phase": 5
 		},
+		"customAnnotationSyntaxInTheTextFormat": {
+			"description": "Custom Text Format Annotations",
+			"url": "https://github.com/WebAssembly/annotations/blob/main/proposals/annotations/Overview.md",
+			"phase": 5
+		},
 		"customPageSizes": {
-			"description": "Custom page sizes",
+			"description": "Custom Page Sizes",
 			"url": "https://github.com/WebAssembly/custom-page-sizes/blob/main/proposals/custom-page-sizes/Overview.md",
-			"phase": 1
+			"phase": 2
+		},
+		"esmIntegration": {
+			"description": "ESM Integration",
+			"url": "https://github.com/WebAssembly/esm-integration",
+			"phase": 3
+		},
+		"exceptionsFinal": {
+			"description": "Exception Handling",
+			"url": "https://github.com/WebAssembly/exception-handling/blob/master/proposals/exception-handling/Exceptions.md",
+			"phase": 4
 		},
 		"exceptions": {
-			"description": "Exception handling",
+			"description": "Exception Handling",
```

Comment:
How about we call this "Legacy Exception Handling" to match the spec, and for now how about calling `exceptionsFinal` "Exception Handling with exnref` to disambiguate and for consistency with Emscripten. Once some time has passed and the emscripten/LLVM implementation is mature we can start changing the language in the documentation to make exnref more like the "default".

Reply:
Alright, I renamed them.